docs/pom.xml

```diff
@@ -76,6 +76,11 @@
             <artifactId>quarkus-langchain4j-watsonx</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
@geoand I wasn't sure if I needed to do this in order for the easy-rag configuration adocs to get generated?

Reply:
Yeah, IIRC you do need it